osu.Game/Screens/Play/MasterGameplayClockContainer.cs

```diff
@@ -112,6 +107,19 @@ protected override void LoadComplete()
             Seek(startOffset);
         }
 
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            userAudioOffset = config.GetBindable<double>(OsuSetting.AudioOffset);
+            userAudioOffset.BindValueChanged(offset => userGlobalOffsetClock.Offset = offset.NewValue, true);
+
+            beatmapOffsetSubscription = realm.SubscribeToPropertyChanged(
+                r => r.Find<BeatmapInfo>(beatmap.BeatmapInfo.ID)?.UserSettings,
+                settings => settings.Offset,
+                val => userBeatmapOffsetClock.Offset = val);
```

Comment:
At this point this no longer feels like a hotfix. Maybe we'll just leave it broken until the proper fix is applied.

Reply:
Alternatively you can schedule the `NonFrameStableSeek`, I think that could work.

Reply:
and undo the movement in this PR, right?

Reply:
I would hope so, since the `NonFrameStableSeek` and `GameplayClockContainer.LoadComplete` calls are only one frame apart, so you only need a `ScheduleAfterChildren` call (you can also expose `GameplayClockContainer`'s `ScheduleAfterChildren` method and queue the first frame bundle seek there for extra safety).

Reply:
Have applied the change, seems to work in limited testing.